package.json

```
@@ -141,6 +141,20 @@
           },
           "scope": "resource"
         },
+        "rubyTestExplorer.debugger": {
+          "description": "Which debugger to use. `rdebug-ide` or `rdbg`.",
```

This should probably be `markdownDescription` given the use of backticks, otherwise they won't be rendered nicely
```suggestion
          "markdownDescription": "Which debugger to use. `rdebug-ide` or `rdbg`.",
```

---

also need to add ruby_lsp